database/models/Event.js

```diff
@@ -0,0 +1,62 @@
+const Sequelize = require('sequelize')
+
+module.exports = class Event extends Sequelize.Model {
+  static init(sequelize) {
+    return super.init(
+      {
+        id: {
+          type: Sequelize.STRING,
+          unique: true,
+          primaryKey: true,
+        },
+        name: {
+          type: Sequelize.STRING,
+          unique: true,
+        },
+        location: {
```

Comment:
I think the location should be an object and not a string since we will store both the longitude and lattitude.

Reply:
Thinking about it, I think it makes sense.

Reply:
- How about dealing with addresses only
- Every event holding place should have a valid address for easy finding
- So, instead of complicating the whole thing with latitude and longitude, we use Google's address system to our advantage

Reply:
We can keep it simple for the MVP sake but we will need to rethink it in future @brianmugweru .

Reply:
sequelize supports geospatial data in form of Geometry, `Sequelize.GEOMETRY` which can be used to store the longitude and latitude of the location. 

I too think we should keep it simple for now. then when we start implementing the front end, and integrate maps, then we can always write a migration to alter the table accordingly. Maybe add a field that will hold the location coordinates.